src/scripts/main.js

```diff
@@ -1,3 +1,9 @@
 'use strict';
 
-// write code here
+const wall = document.querySelector('.wall');
+const spider = document.querySelector('.spider');
+
+wall.style.cssText = 'display: flex; align-items: center;';
+
+spider.style.cssText = 'margin-left: auto; margin-right: auto;'
```

Comment:
```suggestion
wall.style.cssText = 'display: flex; align-items: center;';

spider.style.cssText = 'margin-left: auto; margin-right: auto;'
```
Як сказано у описі задачі
>Yes, it task can be easily done with pure CSS, but not now. Do it with JS.

Ти використовуєш css. Давай обійдемось без цього і використаємо DOM властивості